### ISSUE TITLE
Incorrect redirect to or in wayback

### ISSUE THREAD

**Issue by ldbiz** [ukwa/ukwa-ui#210]
**tl;dr:**

Certain redirects (possibly language related) go to 404s, presumably because a url has been wrongly clipped somewhere.

Displayed in UKWA-UI (and correctly renders when manually entered in the browser url field or if you click on the link here):
https://www.webarchive.org.uk/wayback/en/archive/20150707034232/http://cssiw.org.uk/find-a-care-service/service-directory/15517?lang=cy

Actual redirect when we get to wayback, results in 404, note lang param at end has been clipped.
https://www.webarchive.org.uk/wayback/en/archive/20150707034232/http://cssiw.org.uk/find-a-care-service/service-directory/15517

**Comment by ldbiz** [comment]
tl;dr:

Certain redirects (possibly language related) go to 404s, presumably because a url has been wrongly clipped somewhere.

Displayed in UKWA-UI (and correctly renders when manually entered in the browser url field):
https://www.webarchive.org.uk/wayback/en/archive/20150707034232/http://cssiw.org.uk/find-a-care-service/service-directory/15517?lang=cy

Actual redirect when we get to wayback, results in 404, note lang param at end has been clipped.
https://www.webarchive.org.uk/wayback/en/archive/20150707034232/http://cssiw.org.uk/find-a-care-service/service-directory/15517

**Comment by ldbiz** [comment]
This may be a language issue, as the example is. It probably isn't an UKWA-UI issue, but is raised here for investigation, or reassignment by @anjackson 

See also #27 and #116 which may be related.

If use the following search url (open Access only) 
https://www.webarchive.org.uk/en/ukwa/search?view_filter=va&content_type=Web+Page&from_date=&to_date=&modal_filter_domains_vals=&modal_filter_suffix_vals=&modal_filter_documenttypes_vals=&modal_filter_collections_vals=&filter_source=1&filter_array_x=&filter_array_x_item=&search_location=full_text&text=wiggly+weetabix&view_sort=relevant&view_count=50

and I click on the following link in UKWA

![image](https://user-images.githubusercontent.com/16339290/48428045-12061580-e762-11e8-810b-eb8f18856235.png)

The link below is indicated for the redirect (hover over, see status bar in Chrome):

https://www.webarchive.org.uk/wayback/en/archive/20150707034232/http://cssiw.org.uk/find-a-care-service/service-directory/15517?lang=cy

However when we actually get into wayback (pywayback) the language parameter ("lang=cy") is no longer there, and the page is not found.

![image](https://user-images.githubusercontent.com/16339290/48428296-a1abc400-e762-11e8-9651-cdc10af9c421.png)

This is because the actual redirect is to:

https://www.webarchive.org.uk/wayback/en/archive/20150707034232/http://cssiw.org.uk/find-a-care-service/service-directory/15517

Manually suffixing the param correctly renders the page in wayback.

Somewhere it is going wrong, posibly one of:

1. UKWA-UI is incorrectly clipping the url
2. A config value is incorrect (the referenced issues talk about archive-cy for instance).
3. py-wayback is incorrectly clipping the url.
4. Some other code, indexing or config issue.

**Comment by anjackson** [comment]
This seems to be fine now.
